docs/lib/DEBUGGING.md

```diff
@@ -144,6 +144,20 @@ The OpenSDA port provides a debugger interface, which is compatible with the Seg
 
 To use the JTAG port instead, download the tools and drivers for the debugger of your choice.
 
+#### ULINK pro configuration for NXP FRDM-K64F
+
+Configure NXP FRDM-K64F according to the pdf document from arm Keil [Application Note](http://www.keil.com/appnotes/docs/apnt_287.asp).
+
+In order to add an example to uVision run the "mbed export -i UVISION5 -m K64F" command, and open the project from uVision with the .uvprojx file created.
+For more information check [Exporting to uVision](https://developer.mbed.org/handbook/Exporting-to-uVision).
+
+Most ULINK pro features configuration are described in the Application Note except Instruction trace that is [not officially supported](http://www.keil.com/support/man/docs/ulinkpro/ulinkpro_trace_instruction_trace.htm) with cortex-M devices, we can bypass that with the PC SAMPLING option (as described in the image below)
+
+![](../img/ULINKpro.png)
+
+Choose a Prescaler so that Period will be shorter than core clock time (1/frequency).
+Core clock frequency is set to 120MHz so the core clock time is 8.3us, the Period should be shorter than 8.3us (Prescaler of 64x15 should provide a Period of 8us), Mark the “Periodic” and approve.
```

Comment:
- For proper rendering, please add a line between this one and the one above to start a new paragraph, or delete the enter to keep them in the same paragraph.
- Please change to "The core clock frequency is set to 120MHz, so the core clock time is 8.3us. The period should be shorter than 8.3us (prescaler of 64x15 provides a period of 8us). Mark `Periodic`, and approve."

Reply:
Fixed all, thanks!